packages/react/src/utils/css-state.ts

```diff
@@ -1,34 +1,48 @@
-import { css, FlattenInterpolation, ThemeProps } from 'styled-components';
 import { ResolvedTheme } from '../themes/theme';
 
+type FocusType = 'focus' | 'focus-visible' | 'focus-within';
+
+export interface FocusOptions {
+    selector?: string;
+    focusTypeSelector?: FocusType;
```

Comment:
Je le racourcisserais, c'est assez clair de quoi on parle.

```suggestion
    focusType?: FocusType;
```